pkg/api/routes.go

```diff
@@ -184,17 +185,17 @@ func (rh *RouteHandler) ListTags(response http.ResponseWriter, request *http.Req
 			return
 		}
 
-		var n1 int64
+		var nQuery1 int64
```

Comment:
look at my original PR, pretty sure fixed all these

Reply:
my branch derived from yours, which didn't have any change on this line https://github.com/rchincha/zot/blob/6aa3038c6366030265401f75eb3e2d72514427c9/pkg/api/routes.go#L192